### PR TITLE
Add URL node address display

### PR DESCRIPTION
**Pull request by nikitayutanov**
Provides node address as a URL parameter

### CODE REVIEW

File: website/frontend/src/api/initApi.ts [hunk 1]
```diff
@@ -20,15 +32,15 @@ class NodeApi {
   readonly subscriptions: Record<string, UnsubscribePromise> = {};
 
   constructor(address = 'ws://localhost:9944') {
```
Comment (codev0) [inline website/frontend/src/api/initApi.ts:34]: How about to move getting this one from consts?

File: website/frontend/src/components/blocks/Header/Sidebar/Sidebar.tsx [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+import { useHistory, useLocation } from 'react-router-dom';
+import { NODE_ADRESS_URL_PARAM } from 'consts';
+import * as init from './init';
+import './Sidebar.scss';
+
+type Props = {
+  closeSidebar: () => void;
+};
+
+const Sidebar: VFC<Props> = ({ closeSidebar }) => {
```
Comment (codev0) [inline website/frontend/src/components/blocks/Header/Sidebar/Sidebar.tsx:22]: I think Sidebar is not best name for this kind of component, you could have a look at [mui](https://mui.com/components/drawers/) or [bootstrap](https://getbootstrap.com/docs/5.0/components/offcanvas/).

File: website/frontend/src/components/blocks/Header/Sidebar/init.ts [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+    id: 1,
+    caption: 'test network',
+    nodes: [{ id: 1, custom: false, isChoose: false, address: 'wss://rpc-node.gear-tech.io:443' }],
+  },
+  {
+    id: 2,
+    caption: 'development',
+    nodes: [{ id: 2, custom: false, isChoose: false, address: 'ws://localhost:9944' }],
+  },
+];
```
Comment (codev0) [inline website/frontend/src/components/blocks/Header/Sidebar/init.ts:12]:
```suggestion
] as const;
```

File: website/frontend/src/helpers.ts [hunk 4]
```diff
@@ -57,3 +57,8 @@ export const signPayload = async (injector: any, address: string, payload: any,
     console.error(err);
   }
 };
+
+export const isNodeAddressValid = (address: string) => {
+  const addressRegex = /(ws|wss):\/\/[\w-.]+/gm;
```
Comment (codev0) [inline website/frontend/src/helpers.ts:62]: we can move all regexes to certain place

File: website/frontend/src/components/blocks/Header/Sidebar/init.ts [hunk 5]
```diff
@@ -0,0 +1,12 @@
+export const nodes = [
+  {
```
Comment (codev0) [inline website/frontend/src/components/blocks/Header/Sidebar/init.ts:2]: I think you could type this structure

File: website/frontend/src/components/blocks/Header/Sidebar/Sidebar.tsx [hunk 6]
```diff
@@ -0,0 +1,180 @@
...
+
+  const handleAddNode = () => {
+    const nodeToAdd = {
+      id: nodes[1].nodes.length + 2,
+      custom: true,
+      isChoose: false,
+      address: newNode,
+    };
+
+    setNodes((elems: any) =>
```
Comment (codev0) [inline website/frontend/src/components/blocks/Header/Sidebar/Sidebar.tsx:44]: could you implement this one without any?

File: website/frontend/src/components/App/App.tsx [hunk 7]
```diff
@@ -88,52 +101,52 @@ const AppComponent: FC = () => {
...
-    </BrowserRouter>
+        </Main>
+        {isFooterHidden() || <Footer />}
+        <Alert />
+      </div>
+    </AlertProvider>
   );
 };
 
 export const App = () => (
```
Comment (codev0) [inline website/frontend/src/components/App/App.tsx:146]: Could you refactor this in the way one component per file?

**Review by codev0 — CHANGES_REQUESTED** [review]

File: website/frontend/src/components/App/App.tsx (hunk 7 above)
Comment (nikitayutanov) [reply website/frontend/src/components/App/App.tsx:146]: https://app.clickup.com/t/21zuycw

File: website/frontend/src/components/blocks/Header/Sidebar/Sidebar.tsx (hunk 6 above)
Comment (nikitayutanov) [reply website/frontend/src/components/blocks/Header/Sidebar/Sidebar.tsx:43]: It's a standalone task.

File: website/frontend/src/api/initApi.ts (hunk 1 above)
Comment (nikitayutanov) [reply website/frontend/src/api/initApi.ts:34]: It's a standalone task.

File: website/frontend/src/components/blocks/Header/Sidebar/init.ts (hunk 5 above)
Comment (nikitayutanov) [reply website/frontend/src/components/blocks/Header/Sidebar/init.ts:2]: It's a standalone task.

File: website/frontend/src/components/blocks/Header/Sidebar/Sidebar.tsx (hunk 2 above)
Comment (nikitayutanov) [reply website/frontend/src/components/blocks/Header/Sidebar/Sidebar.tsx:22]: Sidebar is chosen by a collective decision.

**Comment by nikitayutanov** [comment]
All resolved conversations without changes should be a standalone tasks.

**Review by osipov-mit — APPROVED** [review]

**Review by aderrod94 — APPROVED** [review]

**Review by codev0 — APPROVED** [review]

**Pull request merged**